lightning/src/ln/channel.rs

```diff
@@ -4197,6 +4196,15 @@ impl<Signer: Sign> Channel<Signer> {
 		);
 	}
 
+	/// Allowed in any state (including after shutdown)
+	pub fn get_counterparty_htlc_max_msat(&self) -> u64 {
```

Comment:
I think this should return an `Option` and return `None` if we haven't received counterparty info yet (ie their `accept_channel` in an outbound channel.

Reply:
So check if the channel `is_live`, `is_usable`, or `is_outbound` and if not return `None`?

Reply:
Hmm, dunno if we need to wait that long, we can just check that `self.channel_state >= ChannelState::TheirInitSent as u32`